MicrosoftStoreServicesSample/Controllers/CollectionsController.cs

```diff
@@ -109,6 +109,18 @@ public async Task<ActionResult<string>> Query([FromBody] ClientCollectionsQueryR
                 return response.ToString();
             }
 
+            bool includeJson = false;
+            if (Request.Headers.ContainsKey("User-Agent"))
+            {
+                string userAgent = Request.Headers["User-Agent"];
+                if (!string.IsNullOrEmpty(userAgent) && userAgent.Equals("Microsoft.StoreServicesClientSample"))

```

Comment:
Prefer == to .Equals unless you need a case-insensitive compare.

Reply:
OK